x-pack/platform/plugins/shared/lens/public/editor_frame_service/editor_frame/editor_frame.test.tsx

```diff
@@ -215,16 +218,18 @@ describe('editor_frame', () => {
       const { store } = renderEditorFrame();
       const updatedState = 'updatedVisState';
 
-      store.dispatch(
-        setState({
-          visualization: {
-            activeId: mockVisualization.id,
-            state: updatedState,
-          },
-        })
-      );
+      act(() => {
+        store.dispatch(
+          setState({
+            visualization: {
+              activeId: mockVisualization.id,
+              state: updatedState,
+            },
+          })
+        );
+      });
 
-      expect(mockVisualization.getConfiguration).toHaveBeenCalledTimes(3);
+      expect(mockVisualization.getConfiguration).toHaveBeenCalledTimes(4);
```

Comment:
why this changed?

Reply:
@dej611, I haven't looked into these two specific failures. Clearly, this is testing React internals, and I assumed it is what it is, so we can just update. We have two options:  
1. Leave it as is and assume it's fine.  
2. Revert to legacyRoot: true, and your team can investigate these two issues more deeply when migrating the app to concurrent mode.